src/marqo/tensor_search/tensor_search.py

```diff
@@ -2165,6 +2316,10 @@ def vectorise_multimodal_combination_field(
     try:
         start_time = timer()
         text_vectors = []
+
+        # Add prefix to all text content first
+        text_content_to_vectorise = text_processor.prefix_text_chunks(text_content_to_vectorise, text_chunk_prefix)
```

Comment:
Minor but best to call this something else rather than overwrite/reuse existing variable

Reply:
done